subsys/bluetooth/mesh/gen_plvl_srv.c

```diff
@@ -628,7 +628,7 @@ static void scene_recall_complete(struct bt_mesh_model *model)
 	(void)bt_mesh_plvl_srv_pub(srv, NULL, &status);
 }
 
-/*  MeshMDL1.0.1, section 5.1.3.1.1:
+/*  MeshMDLv1.1: 5.1.3.1.1:
```

Comment:
I think the abbreviation used by BT SIG is `MshMDL`? I think we either should use that or the full name, not this almost-the-same abbreviation.

Reply:
I see we also randomly use one of the two in various locations in this PR. Please unify this so that every reference is `MshMDL`.

Reply:
My bad, It is supposed to be `MshMDL`.